local-tests/setup/shiva-client.ts

```diff
@@ -220,21 +240,9 @@ export class ShivaClient {
   }
 
   /**
-     * Used to start an instance of a lit network through the Lit Testnet Manager
-     * if an isntance exists, we will just take it as we optimistically assume it will not be shut down in the test life time.
-     * If an instance does not exist then we create one
-     * struct reference
-        pub struct TestNetCreateRequest {
-        pub node_count: usize,
-        pub polling_interval: String,
-        pub epoch_length: i32,
-        pub custom_build_path: Option<String>,
-        pub lit_action_server_custom_build_path: Option<String>,
-        pub existing_config_path: Option<String>,
-        pub which: Option<String>,
-        pub ecdsa_round_timeout: Option<String>,
-        pub enable_rate_limiting: Option<String>,
-       }
+   * Used to start an instance of a lit network through the Lit Testnet Manager
+   * if an isntance exists, we will just take it as we optimistically assume it will not be shut down in the test life time.
```

Comment:
:nit: typo
```suggestion
   * if an instance exists, we will just take it as we optimistically assume it will not be shut down in the test life time.
```